dist/cms.js

```diff
@@ -35,10 +35,52 @@ function getWordPressEmbedBlockCounts() {
   }
 }
 
+/**
+ * Obtains data about scripts that WordPress prints on the page.
+ *
+ * @returns {{}[]}
+ */
+function getWordPressScripts() {
+  const entries = [];
+
+  /**
+   * Checks whether the provided element is an inline script.
+   *
+   * @param {?Element} element Element to examine. May be null.
+   * @return {?number} Size of the inline script or null if the element isn't an inline script.
+   */
+  const getInlineScriptSize = (element) => {
+    if (element instanceof HTMLScriptElement && !element.src) {
+      return element.textContent.length;
+    }
+    return null;
+  };
+
+  const scripts = document.querySelectorAll('script[src][id$="-js"]');
+  for (const script of scripts) {
+    /** @var HTMLScriptElement script */
+    const handle = script.id.replace(/-js$/, '');
+    entries.push({
+      handle,
+      src: script.src,
+      in_footer: script.parentNode !== document.head,
```

Comment:
This doesn't imply it's in the footer. Just that it's in the body. Should the name be changed?

```suggestion
      in_body: script.parentNode !== document.head,
```

Reply:
The reason for using `in_footer` is that this is the parameter name used in the WordPress [`wp_register_script`](https://developer.wordpress.org/reference/functions/wp_register_script/)/[`wp_enqueue_script`](https://developer.wordpress.org/reference/functions/wp_enqueue_script/) APIs. It is in the head if `script.parentNode === document.head`, so the opposite of that `!==` means it is `in_footer`.

Reply:
OK. Though this script could also pick up other scripts not inserted by that. Anyway, happy enough to go with `in_footer` anyway since it's in a `wordpress` block.